### PR TITLE
Add scripts directory and checkout-pr script for testing pull requests

### PR DESCRIPTION
**Pull request by MaxLeiter**
Short Description: Add a `scripts/` directory for future development scripts. Added a `checkout-pr.sh` script for maintainers to easily test pull requests.

Changes:
- Add `scripts/` directory
- Add `scripts/checkout-pr.sh` file

@OpenPoGo/maintainers

### CODE REVIEW

File: scripts/run-pr.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+set -e
+
+if [ -z "$1" ]; then
+  echo "No pull request ID was specified."
+  exit 1
+fi
+
+git fetch https://github.com/OpenPoGo/OpenPoGoBot.git refs/pull/${1}/head
+git checkout FETCH_HEAD
+npm install
```
Comment (jtdroste) [inline scripts/run-pr.sh]: npm?

**Comment by jmeyer2k** [comment]
:+1:

**Comment by jtdroste** [comment]
👎 

[Rejected with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

File: scripts/run-pr.sh (hunk 1 above)
Comment (MaxLeiter) [inline scripts/run-pr.sh]: nice catch, was rushing.

**Comment by MaxLeiter** [comment]
Ready for a second review, @jtdroste

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jtdroste** [comment]
_Technically_ your commit is referencing the wrong file :P

👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
Fixed commit message; ready for review :) (remind that the :+1: must be at the start of your message, I think)

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by wchill** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**